langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -99,6 +117,21 @@ public isolated function get(xml<ItemType> x, int i) returns ItemType = @java:Me
 } external;
 
 # Concatenates xml and string values.
+# 
+# ```ballerina
+# xml bookA = xml `<book>Ballerina</book>`;
+# xml bookB = xml `<book>Java</book>`;
+# xml:concat(bookA, bookB, xml `<book>Python</book>`) ⇒ <book>Ballerina</book><book>Java</book><book>Python</book>
```

Comment:
The guidelines state to use method call syntax over function call syntax right, so shall we avoid using `xml:concat`

Reply:
There are examples provided in the method call syntax. 
The approach used here is similar to: https://github.com/ballerina-platform/ballerina-lang/pull/39036/files#diff-2aea12d058828f6e61ef836e0d288028338e5bcc564514b55cdc0fe095bf7b6cR63 as per the latest discussions held.